index.html

```diff
@@ -4,14 +4,28 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <title>Aaron Keegan Developer Portfolio</title>
 </head>
 <body>
-
-  <!-- header
-  name:Aaron, sandwich icon -->
-  <!-- Section h1, h3, p (about) icons,
-  background image -->
-  
+  <header>
+    <nav class="top-nav">
+      <div class="logo">Aaron Keegan</div>
+      <div class="contact">&#x2630;</div>
+    </nav>
+  </header>
+  <section class="welcome">
+    <h1 class="greeting">Hey There. I'm Aaron</h1>
+    <h3 class="who-am-i">I am a software Developer</h3>
+    <p>Welcome to my portfolio page. My goal is to help you expand your business or share ideas around the globe. The power of the internet makes that a reality today! 
+      For info about me and for snapshots of my work, please continue reading my site.</p>
+    <div class="contact-bar">
+      <a href="#" class="fa fa-facebook"></a>
+      <a href="#" class="fa fa-twitter"></a>
+      <a href="#" class="fa fa-linkedin"></a>
+      <a href="#" class="fa fa-instagram"></a>
+    </div>
```

Comment:
Due to semantic reasons, it's a good practice to wrap these social icons inside an unordered list ul tag 🤓
Kindly perform these suggestions so that the list of icons uses a ul tag to group them, and li per icon!